lib/engines/content_object_store/test/factories/content_block_edition.rb

```diff
@@ -22,8 +22,10 @@
     end
 
     after(:create) do |content_block_edition, evaluator|
-      document = create(:content_block_document, evaluator.content_block_document_attributes)
-      content_block_edition.update!(content_block_document_id: document.id)
+      unless content_block_edition.content_block_document_id
```

Comment:
When we want to create an Edition and assign our own Document to it then we need a way to avoid creating duplicate Documents.